_episodes/06-feedback.md

```diff
@@ -57,44 +57,44 @@ a dashboard with the results! Take care not to share this link with your learner
 ![Screenshot of a workshop website showing location of customized survey links](../fig/surveyscreenshot3.svg)
 
 ### Timing Matters
- 
+
 We have found that learners are much more likely to fill out the post-workshop survey
-while they are still at the workshop than they are after they leave the venue. At the end 
-of a two-day workshop, your learners' brains will be very tired. Rather than trying to 
+while they are still at the workshop than they are after they leave the venue. At the end
+of a multi-day workshop, your learners' brains will be very tired. Rather than trying to
 fit in another 15 minutes worth of teaching, give your learners time to complete the
 post-workshop survey at the end of your workshop. You will be helping them (with an
-opportunity to reflect), yourself (you will get more useful feedback), and The Carpentries 
+opportunity to reflect), yourself (you will get more useful feedback), and The Carpentries
 (we improve our programs based on feedback, and our funders take pride in the success of our workshops, too).
 
 
 ## Minute Cards
 
-Before each lunch break and at the end of the first day, we
-have learners complete **minute cards** to share anonymous feedback. At an in-person workshop, 
-paper sticky notes double as minute cards, with the two different colors used for positive and 
+During any lunch breaks and between days, we
```

Comment:
```suggestion
Before each long break, for example lunch or between days, we
```
I mostly want to keep the "before" not "during" while it may in practice sometimes bleed into break time, we should at least try to advise people to do this before the break time starts.